### ISSUE TITLE
update RPM dependency for EL8

### ISSUE THREAD

**Issue by balonik** [rancher/k3s-selinux#23]
Hi,

EL8 (CentOS) is already using newer `container-selinux` package -> [link](https://centos.pkgs.org/8/centos-appstream-x86_64/container-selinux-2.167.0-1.module_el8.4.0+942+d25aada8.noarch.rpm.html)
Could you please update the dependency in spec file?

Thanks.

**Comment by Oats87** [comment]
The reason we have capped EL8 support is due to conflicting file contexts that were merged upstream. As such, we're dealing with needing to reconcile against these conflicting file contexts.

The capping for the `container-selinux` is done here: https://github.com/k3s-io/k3s-selinux/pull/21

The conflicting file contexts were merged here: https://github.com/containers/container-selinux/pull/140

@dweomer I think at some point it is likely reasonable for us to amend our policy for `EL8` to comment out the conflicting upstream policies, but I do worry about breaking backwards compatibility for our users who are not willing to update to the latest set of RPMs.

**Comment by larsks** [comment]
I think this is the same problem.

I've just upgraded from Fedora 34 to 35, and I see:

```
 Problem: package container-selinux-2:2.170.0-2.fc35.noarch conflicts with k3s-selinux <= 0.4-1 provided by k3s-selinux-0.2-1.el7_8.noarch
  - cannot install the best update candidate for package container-selinux-2:2.169.0-1.fc35.noarch
  - problem with installed package k3s-selinux-0.2-1.el7_8.noarch
```

Or when trying to install k3s-selinux:

```
 Problem: conflicting requests
  - nothing provides container-selinux < 2:2.164.2 needed by k3s-selinux-0.4-1.el8.noarch
```

**Comment by zdzichu** [comment]
@larsks you have `k3s-selinux 0.4-1`, try with https://github.com/k3s-io/k3s-selinux/releases/tag/v0.5.testing.1

**Comment by larsks** [comment]
@zdzichu thanks, that installs without the conflict, at least. Nothing has broken yet, so it seems to be working :).

**Comment by shane-axiom** [comment]
Apologies if I'm threadjacking here, but I think I'm hitting the same while trying in to bake k3s-selinux into a Fedora CoreOS image:

```
error: Could not depsolve transaction; 1 problem detected:
 Problem: conflicting requests
  - package container-selinux-2:2.170.0-2.fc35.noarch conflicts with k3s-selinux <= 0.4-1 provided by k3s-selinux-0.2-1.el7_8.noarch
  - package container-selinux-2:2.170.0-2.fc35.noarch conflicts with k3s-selinux <= 0.4-1 provided by k3s-selinux-0.3-0.el7.noarch
  - package moby-engine-20.10.9-1.fc35.x86_64 requires container-selinux, but none of the providers can be installed
  - package k3s-selinux-0.4-1.el7.noarch requires container-selinux < 2:2.164.2, but none of the providers can be installed
  - package k3s-selinux-0.5-1.el7.noarch requires container-selinux < 2:2.164.2, but none of the providers can be installed
  - package container-selinux-2:2.158.0-1.gite78ac4f.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.160.0-1.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.160.0-2.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.162.1-3.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.162.2-1.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.163.0-1.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.164.1-1.git563ba3f.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.164.2-1.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.164.2-1.fc35.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.167.0-1.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.168.0-1.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.168.0-1.fc35.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.169.0-1.fc35.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.170.0-1.fc34.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.170.0-1.fc35.noarch is filtered out by exclude filtering
  - package container-selinux-2:2.170.0-2.fc34.noarch is filtered out by exclude filtering
```

Looks like its trying to use `k3s-selinux-0.5-1.el7.noarch` but `container-selinux` is too new?

**Comment by dweomer** [comment]
Fixed via #24 and https://github.com/k3s-io/k3s-selinux/releases/tag/v0.5.stable.1
